docs/pages/load-testing/results.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Test results
+sidebar_label: Results
+---
+
+## Summary
+
+This is a TL;DR of the test results, the detailed results are further down. During our testing, k3s with sqlite has been trailing the other distributions when running high intensity loads. For less intensive usage however, it has been a little slower than the others but staying well within the usable range. The simplicity of its deployment comes with a few downsides.
+If you plan on having high api usage in your vClusters, we recommend using an etcd backed distribution as you will most likely experience timeouts or throttling with the sqlite backed distribution. For lesser usage, the k3s sqlite will be as adequate as the others.
+
+
+## API requests response time
+
+<figure>
+  <img src="/docs/media/diagrams/apiserver-latency-baseline.svg" alt="apiserver-avg-baseline" />
+  <figcaption>APIserver average response time (baseline)</figcaption>
+</figure>
+
+During our baseline testing (300 secrets, 30qps), the k3s with sqlite has been significantly slower than the other distributions, with an average of 0.17s while the other distributions were all around 0.05s. This however should not have an impact since 0.17 is still a relatively good average.
```

Comment:
```suggestion
During our baseline testing (300 secrets, 30qps), K3s with SQLite was significantly slower than the other distributions, with an average of 0.17s while the other distributions were all around 0.05s. This however should not have an impact since 0.17 is still a relatively good average.
```